addons/project/models/project.py

```diff
@@ -269,6 +269,17 @@ def _compute_alias_enabled(self):
         for project in self:
             project.alias_enabled = project.alias_domain and project.alias_id.alias_name
 
+    def _inverse_privacy_visibility(self):
```

Comment:
As this method is will no more be provided to an inverse attribute of a compute field I would rename in order not to have it starting with _inverse_xxx.

Reply:
Done ! Thanks !